front/lib/api/invitation.ts

```diff
@@ -15,7 +15,7 @@ export async function sendWorkspaceInvitationEmail(
   // Create MembershipInvitation.
   const invitation = await MembershipInvitation.create({
     workspaceId: owner.id,
-    inviteEmail,
+    inviteEmail: inviteEmail.toLowerCase(),
```

Comment:
Should we implement a function sanitizeEmail that would do both lowerCase and trim?

Reply:
I guess we should just have a generic sanitizer